packages/wallet/backend/src/rapyd/controller.ts

```diff
@@ -81,6 +90,36 @@ export class RapydController implements IRapydController {
       req.session.user.needsWallet = false
       await req.session.save()
 
+      if (this.deps.env.NODE_ENV === 'production') {
+        const asset = (await this.deps.rafikiClient.listAssets()).find(
```

Comment:
I suggest to extract this into a `createDefaultAccount` method

Reply:
Having it inside the `rapydService` wouldn't make sense. I abstracted away the `createDefaultAccount` method. The payment pointer creation does not happen in this method because we will end up with circular dependencies.